geometry_msgs/Pose curr_pose
geometry_msgs/Pose cylinder_pose
string color
string qr_data
